### PR TITLE
Created profile for gcp e2e test run excluding billing metrics test c…

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
Created profile for gcp e2e test run excluding billing metrics test classes

### CODE REVIEW

File: suites/gcp.xml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        <classes>
+            <class name="io.managed.services.test.SSOAuthTest"/>
+        </classes>
+    </test>
+    <test name="TestBase">
+        <classes>
+            <class name="io.managed.services.test.TestBase"/>
+        </classes>
+    </test>    
+    <test name="DeploymentsTest">
```
Comment (henryZrncik) [inline suites/gcp.xml:19]: reserved deployment will not be running against GCP without mod. this class first

File: suites/gcp.xml [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    <test name="KafkaScriptsSteps">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.steps.KafkaScriptsSteps"/>
+        </classes>
+    </test>
+    <test name="KafkaTopicSteps">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.steps.KafkaTopicSteps"/>
+        </classes>
+    </test>
```
Comment (henryZrncik) [inline suites/gcp.xml:114]: there are no tests in these context, why are they here ?  (same for those below)

File: suites/gcp.xml (hunk 1 above)
Comment (tonyxrmdavidson) [reply suites/gcp.xml]: Removed Deployments Test

File: suites/gcp.xml (hunk 2 above)
Comment (tonyxrmdavidson) [reply suites/gcp.xml]: Removed all test with .contexts and .steps

File: suites/gcp.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        <classes>
+            <class name="io.managed.services.test.kafka.LongLiveKafkaInstanceTest"/>
+        </classes>
+    </test>
+    <test name="QuotaKafkaInstanceTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.QuotaKafkaInstanceTest"/>
+        </classes>
+    </test>
+    <test name="CucumberQuickstartsTest">
```
Comment (henryZrncik) [inline suites/gcp.xml:69]: we do not execute this quickstart before devExp team resolve issue with token ivnalidaiton.

File: suites/gcp.xml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    <test name="SSOAuthTest">
+        <classes>
+            <class name="io.managed.services.test.SSOAuthTest"/>
+        </classes>
+    </test>
+    <test name="TestBase">
+        <classes>
+            <class name="io.managed.services.test.TestBase"/>
+        </classes>
+    </test>
```
Comment (henryZrncik) [inline suites/gcp.xml:18]: No need.

File: suites/gcp.xml [hunk 5]
```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="GCP">
+    <test name="SmokeTest">
+        <classes>
+            <class name="io.managed.services.test.SmokeTest"/>
+        </classes>
+    </test>
```
Comment (henryZrncik) [inline suites/gcp.xml:8]: Not needed.

**Review by henryZrncik — CHANGES_REQUESTED** [review]

File: suites/gcp.xml [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    <test name="CucumberQuickstartsTest">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.CucumberQuickstartsTest"/>
+        </classes>
+    </test>
+    <test name="QuarkusApplicationTest">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.QuarkusApplicationTest"/>
+        </classes>
+    </test>
```
Comment (henryZrncik) [inline suites/gcp.xml:74]: will intrfere with tokens untill we resolve it in CLI (so it should be excluded up until then)
```suggestion
    <test name="QuarkusApplicationTest">
        <classes>
            <class name="io.managed.services.test.quickstarts.QuarkusApplicationTest"/>
        </classes>
    </test>
```

Comment (henryZrncik) [reply suites/gcp.xml:74]: not resolved ...

Comment (tonyxrmdavidson) [reply suites/gcp.xml]: Removed

**Review by henryZrncik — APPROVED** [review]
looks good. 
Just mentioning: Today/Tomorrow there will be a bit of refactoring in CLI, its result will be `KafkaCLITest` and `RhoasCliTest` now under its own package.

**Pull request merged**